packages/website/docs/_components_pages/main/Table/TableCell.mdx

```diff
@@ -3,6 +3,22 @@ slug: ../../TableCell
 sidebar_class_name: newComponentBadge expComponentBadge
 ---
 
+import HAlign from "../../../_samples/main/Table/HAlign/HAlign.md";
+
 <%COMPONENT_OVERVIEW%>
 
 <%COMPONENT_METADATA%>
+
+### Horizontal cell alignment
+
+Control the horizontal alignment of cells by utilizing the `horizontalAlign` property.
+
+Please note that the behaviour of `horizontalAlign` depends on where you set it:
+1. `horizontalAlign` is set on `TableHeaderCell` level<br />
+Changes the horizontal alignment of the `TableHeaderCell` and their corresponding `TableCells`.
```

Comment:
I know I am picky that way, but isn't the technical name /component name , strictly speaking, TableCell ?